client/auth/login.jsx

```diff
@@ -113,7 +111,8 @@ export const Login = createReactClass( {
 									disabled={ requires2fa || inProgress }
 									placeholder={ translate( 'Username or email address' ) }
 									onFocus={ this.recordFocusEvent( 'Username or email address' ) }
-									valueLink={ this.linkState( 'login' ) }
+									value={ this.state.login }
```

Comment:
other note to other reviewers, yes, these usages of `state` are safe. `linked-state-mixin` required component state to work, so if this stuff worked previously, there must have been a `getInitialState` method defined. You don't really have to check every component to make sure it also has a `getInitialState` method.